custom_components/zwave_mqtt/binary_sensor.py

```diff
@@ -36,3 +80,101 @@ class ZWaveBinarySensor(ZWaveDeviceEntity, BinarySensorDevice):
     def is_on(self):
         """Return if the sensor is on or off."""
         return self.values.primary.value
+
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        # Legacy binary sensors are phased out (replaced by notification sensors)
+        # Disable by default to not confuse users
+        return False
+
+
+class ZWaveListSensor(ZWaveDeviceEntity, BinarySensorDevice):
+    """Representation of a ZWaveListSensor translated to binary_sensor."""
+
+    @property
+    def is_on(self):
+        """Return if the sensor is on or off."""
+        return self.values.primary.value["Selected"] != "Clear"
+
+    @property
+    def state_attributes(self):
```

Comment:
```suggestion
    def device_state_attributes(self):
```

Reply:
Hmmm, in that case it needs more work as the base entity already has the device_state_attributes. I was hoping to not override that. What's the big difference between the two anyway ?

Reply:
Each base entity integration, eg components/binary_sensor/__init__.py, implements `state_attributes`. This should not be overridden in the platforms. The platforms can implement `device_state_attributes` which is merged into `state_attributes` when updating state.

Reply:
fixed